wrapper/src/test/java/integration/host/TestEnvironmentProvider.java

```diff
@@ -112,8 +112,8 @@ public Stream<TestTemplateInvocationContext> provideTestTemplateInvocationContex
               // Multi-AZ clusters supports only 3 instances
               continue;
             }
-            if (deployment == DatabaseEngineDeployment.RDS_MULTI_AZ_INSTANCE && numOfInstances != 1) {
-              // Multi-AZ Instances supports only 1 instance
+            if (deployment == DatabaseEngineDeployment.RDS_MULTI_AZ_INSTANCE && numOfInstances > 2) {
```

Comment:
That's not correct. RDS Instance is always 1 instance. Standby reader, if it's presented/deployed, isn't visible through topology or any other ways.